appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/VulnerabilityDetailsView.java

```diff
@@ -213,21 +223,72 @@ private Component buildVaadinAnalysis() {
         vaTitle.addStyleName("vaadin-analysis-title");
         vAnalysisPanel.addComponent(vaTitle);
 
-        // TODO Replace with real data when available
-        Label vaStatus = new Label("Under review");
-        vaStatus.addStyleName("va-under-review");
-        vaStatus.setSizeUndefined();
+        boolean isSupportedVaadinVersion = false;
+        String vaadinVersion = Version.getFullVersion();
+        AppSecService appsec = AppSecService.getInstance();
+        List<String> fw7Versions = appsec.getSupportedFramework7Versions();
+        List<String> fw8Versions = appsec.getSupportedFramework8Versions();
```

Comment:
Why do we need to check for the 8 versions in the `appsec-kit-v7`?

Reply:
Good point, we don't.